src/input/input.jsx

```diff
@@ -23,13 +23,16 @@ const Input = ({
    className, style,
    type, name, label,
    placeholder, disabled,
-   required, options, value,
+   required, options, value, hiddenLabel,
```

Comment:
what about `placeholderToLabel` ?

Reply:
I think the must it's the default value for input not on "Group" it s show placeholder to label.
So i add hiddenLabel only when i'have not need label.